code/modules/mob/living/silicon/robot/robot_modules/station.dm

```diff
@@ -59,13 +59,7 @@ var/global/list/robot_modules = list(
 	handle_shell(R)
 
 	if(R.radio)
-<<<<<<< HEAD
-		if(R.shell)
-			channels = R.mainframe.aiRadio.channels
```

Comment:
Won't this break radio recalculations for AI shells or is it handled elsewhere properly?

Reply:
note the addtimer callback to run the recalculate proc

polaris seems to have been doing just fine without it for a good year or two at least, too